integreat_cms/cms/views/pois/poi_actions.py

```diff
@@ -170,3 +174,36 @@ def view_poi(request, poi_id, region_slug, language_slug):
         raise Http404
 
     return render(request, template_name, {"poi_translation": poi_translation})
+
+
```

Comment:
```suggestion

@json_response
```
Since it's an AJAX view, I suggest to always return a JSON, even on errors